arch/xtensa/soc/esp32/linker.ld

```diff
@@ -36,6 +37,9 @@ MEMORY
   drom0_0_seg(R): org = 0x3F400010, len = 0x800000
   rtc_iram_seg(RWX): org = 0x400C0000, len = 0x2000
   rtc_slow_seg(RW): org = 0x50000000, len = 0x1000
+#ifdef CONFIG_GEN_ISR_TABLES
```

Comment:
unfortunately, we are gonna need to make corresponding changes in all the linker scripts for the other Xtensa SOC definitions, otherwise I do not think we can remove the old IRQ handling code unless we update the configuration for everything